lib/constants.go

```diff
@@ -905,7 +906,7 @@ var DeSoMainnetParams = DeSoParams{
 	// We set this to be lower initially to avoid winding up with really big
 	// spam blocks in the event someone tries to abuse the initially low min
 	// fee rates.
-	MinerMaxBlockSizeBytes: 2000000,
+	MinerMaxBlockSizeBytes: 500000,
```

Comment:
Note that this does not reduce the maximum block size permitted by the network, only the default block size used by miners when generating block templates.